cmd/launcher/options.go

```diff
@@ -178,22 +178,41 @@ func parseOptions(args []string) (*launcher.Options, error) {
 		return nil, err
 	}
 
+	// Set control server URL and control server TLS settings based on Kolide server URL, defaulting to local server
+	controlServerURL := ""
+	insecureTLS := *flInsecureTLS
+	disableControlTLS := false
```

Comment:
Those options do [two different things](https://github.com/kolide/launcher/blob/main/ee/control/client_http_option.go) for the control service client, though -- `insecureTLS` sets `InsecureSkipVerify` on the client, and `disableTLS` sets the base URL's scheme to http.

I agree that it would be better to not overload `*flInsecureTLS` -- how about I add a separate `InsecureControlTLS` option (not controlled by a command line flag) for this?

Reply:
I went ahead and did it ⬆️ this way in the last commit, let me know what you think or if you'd like changes

Reply:
Ah yes, good catch